zerver/lib/actions.py

```diff
@@ -1360,17 +1363,17 @@ def internal_prep_message(sender_email, recipient_type_name, recipients,
 
     try:
         return check_message(sender, get_client("Internal"), recipient_type_name,
-                             parsed_recipients, subject, content, realm)
+                             parsed_recipients, subject, content, realm, sendtonotsubscribed=sendtonotsubscribed)
     except JsonableError as e:
         logging.error("Error queueing internal message by %s: %s" % (sender_email, str(e)))
 
     return None
 
 def internal_send_message(sender_email, recipient_type_name, recipients,
-                          subject, content, realm=None):
-    # type: (Text, str, Text, Text, Text, Optional[Realm]) -> None
+                          subject, content, realm=None, sendtonotsubscribed=False):
+    # type: (Text, str, Text, Text, Text, Optional[Realm], Optional[bool]) -> None
     msg = internal_prep_message(sender_email, recipient_type_name, recipients,
-                                subject, content, realm)
+                                subject, content, realm, sendtonotsubscribed)
```

Comment:
Could you change the variable name to match our style (e.g. `send_to_not_subscribed`)?  I think also maybe we want a name like `internal_notice_to_private_stream`.